simulate.py

```diff
@@ -30,5 +39,7 @@ def simulate_reads(genes,pAi,reads_per_gene=100,pAlen=42,fragment_length=400):
         interval=intervals[0] # pick first 3' UTR isoform for now
         reads[gene]=[]
         for read in range(reads_per_gene):
+            r=random.random()
+            fragment_length=f_size[min([i for i in range(len(f_cum)) if r<=f_cum[i]])]
             reads[gene].append(int(interval['start'])-1+pAlen-fragment_length)
```

Comment:
Could this be a strand issue? If so, please open an issue so I'll add strand awareness to the simulations.